### ISSUE TITLE
Does it work on Windows?

### ISSUE THREAD

**Issue by Triplecorpse** [hex7c0/mongodb-backup#32]
It works perfectly on mac and linux but when I tried to run it on Windows, an error occurred `Error: ENOENT: no such file or directory, open 'C:\Users\eldar\WebstormProjects\romanenkova\backups\2019-01-13T10:57:20.626Z\rom_prod\schedules\5c2cfcc23fb8387050a14958.bson'`

Do I need to set something up or there is no way?

**Comment by Phoscur** [comment]
Yes it does work on windows, please provide more environment information like MongoDB version, mongodb-backup version as well as NodeJS and Windows versions and the options you are using!
